src/backend/cdb/cdbquerycontextdispatching.c

```diff
@@ -801,6 +825,9 @@ RebuildQueryContext(QueryContextInfo *cxt, HTAB **currentFilesystemCredentials,
         	RebuildFilesystemCredentials(cxt, currentFilesystemCredentials,
         	        currentFilesystemCredentialsMemoryContext);
         	break;
+        case Namespace:
+            RebuildNamespace(cxt);
```

Comment:
No, it won't. The `Namespace` enum is only added to the buffer in the case of a secure filesystem